inlong-dashboard/src/ui/components/FieldList/DetailModal.tsx

```diff
@@ -226,6 +223,12 @@ const Comp: React.FC<Props> = ({
     message.success(t('pages.GroupDetail.Sources.SaveSuccessfully'));
   };
 
+  useEffect(() => {
```

Comment:
Can the logic of calling `getData` in several `useEffect` here be processed in one place? (Three `getData` calls were found) Otherwise, under complex conditions, it is easy to cause `getData` in multiple places to be executed, leaving hidden dangers.